client/sites/components/sites-dataviews/style.scss

```diff
@@ -7,16 +7,14 @@
 	overflow: hidden;
 	padding-bottom: 0;
 
-	.sites-dataviews__site {
-		display: flex;
-		flex-direction: row;
-		padding: 0;
-		overflow: hidden;
-		cursor: pointer;
-
-		&:not(:disabled) {
-			&:hover,
-			&:focus {
+	// TODO: DataViews adds .is-hovered to row/li's when hovering over them, we're targeting that
+	// to style the site name with a blue color to match core page list view. We need to target
+	// the row hover as the .sites-dataviews__site button doesn't cover the whole cell.
+	// At some point, this may be able to be cleaned up via a new onClick handler for primary fields,
+	// see https://github.com/WordPress/gutenberg/issues/67391#issuecomment-2506037360
+	.is-hovered {
+		.sites-dataviews__site {
+			.sites-dataviews__site-name:not(.sites-dataviews__site-name--deleted) {
 				.sites-dataviews__site-title {
```

Comment:
In Core, I think the text is not highlighted in the table view when the row is hovered. 
![fb3299ab500271cd2241190434fe0772](https://github.com/user-attachments/assets/bdf48d3c-4683-4004-a9e0-540722426b99)

Reply:
In the list view, it seems to be done with `li:hover .dataviews-view-list__primary-field`: https://github.com/WordPress/gutenberg/blob/ed66cc50e3c0b6785a48c15230c090790c0b0e6c/packages/dataviews/src/dataviews-layouts/list/style.scss#L56-L66. 

Is it possible to remove this code and the color styles we give to the site title and URL, so that only the core style is responsible for the colouring?

Reply:
Oh, we should take care of `:not(.sites-dataviews__site-name--deleted)`